Pages/LoginPage.cs

```diff
@@ -0,0 +1,76 @@
+﻿using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Training_Center_Task_15.Pages
+{
+    public class LoginPage : BasePage
+    {
+        private IWebDriver _driver;
+
+        public LoginPage(IWebDriver driver) : base(driver)
+        {
+            _driver = Driver;
+        }
+        public IWebElement NameNewUser => _driver.FindElement(By.CssSelector("input[data-qa='signup-name']"));
+        public IWebElement EmailAddressNewUser => _driver.FindElement(By.CssSelector("input[data-qa='signup-email']"));
+        public IWebElement SignupButton => _driver.FindElement(By.CssSelector("button[data-qa='signup-button']"));
+        public IWebElement EmailAddressExistingUser => _driver.FindElement(By.CssSelector("input[data-qa='login-email']"));
+        public IWebElement Password => _driver.FindElement(By.CssSelector("input[data-qa='login-password']"));
+        public IWebElement LoginButton => _driver.FindElement(By.CssSelector("button[data-qa='login-button']"));
+
+
+
```

Comment:
also please correct formatting. Remove extra free spaces. Add free spaces after brackets and fields (for example between 17-18 lines)

Reply:
Checked all classes and fixed formatting. Also removed unnecessary usings.